tests/system/test_access.py

```diff
@@ -359,6 +360,36 @@ def upload(token):
             resp = upload(token)
             assert resp.status_code == 202, "token: {}, status_code: {}".format(token, resp.status_code)
 
+@integration_test
+class TestSelfInstrumentationWithAPIKeys(BaseAPIKeySetup):
+
+    def setUp(self):
+        super(TestSelfInstrumentationWithAPIKeys, self).setUp()
+        self.api_key = self.create_api_key([self.privilege_intake], self.resource_any)
+
+    def config(self):
+        cfg = super(TestSelfInstrumentationWithAPIKeys, self).config()
+        cfg.update({"api_key_enabled": True, "enable_rum": True,
+                    "kibana_enabled": "true", "kibana_host": self.get_kibana_url(),
```

Comment:
You only test the event ingestion, setting `enable_rum`, `kibana_enabled` or `kibana_host`  should not be needed here.

Reply:
Copy-pasta :sweat_smile:
Thanks, fixed.